core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -63,10 +66,24 @@ public void testGetNonexistentView() throws Exception {
     statsManager.getView(RpcConstants.RPC_CLIENT_REQUEST_COUNT_VIEW);
   }
 
-  // TODO(sebright): Turn this into a real test once StatsManagerImpl.record(...) is implemented.
-  // It currently only shows that record(...) can be called with a SimpleEventQueue.
   @Test
   public void testRecord() {
+    statsManager.registerView(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    statsManager.record(
+        StatsContextFactoryImpl.DEFAULT,
```

Comment:
I think we will refactor StatsContext so that it is a final class in the core directory.  Then it won't need to be referenced indirectly from StatsContextFactoryImpl.  I made an initial PR: #219